starknet_py/utils/data_transformer/data_transformer.py

```diff
@@ -131,12 +131,18 @@ def to_python(self, cairo_type, name, values) -> (dict, CairoData):
 
 class TupleTransformer(TypeTransformer[TypeTuple, tuple]):
     def from_python(self, cairo_type, name, value):
```

Comment:
Can we include a `return` type because this seems like a classmethod (like a factory) otherwise?

Reply:
Sorry do you mean we should add return types to `from_python` and `to_python`?